charts/incubator/alfresco-dbp/requirements.yaml

```diff
@@ -12,6 +8,14 @@ dependencies:
     condition: alfresco-sync-service.enabled
     repository: https://kubernetes-charts.alfresco.com/incubator
   - name: alfresco-infrastructure
-    version: 4.0.0-SNAPSHOT
+    version: 5.0.0
     condition: alfresco-infrastructure.enabled
     repository: https://kubernetes-charts.alfresco.com/incubator
+  - name: activiti-cloud-full-example
+    version: 1.1.4
+    condition: activiti-cloud-full-example.enabled
+    repository: https://activiti.github.io/activiti-cloud-charts/
+  - name: alfresco-digital-workspace
```

Comment:
Aren't we getting [DW from ACS' chart](https://github.com/Alfresco/acs-deployment/blob/master/helm/alfresco-content-services/requirements.yaml#L20)?

Why are we bringing it again here?

Reply:
Because the DW in the acs chart does not have the identity properties. I think we should really have a story to update ACS and Activiti after this story.

Reply:
I understand setting those properties, but why are we bringing it in as a requirement?

Reply:
Because we bring the newer one that can take those properties.
But you are right, we should pull it just to fix some issue the acs chart has. I'm removing that requirement.